kadmin/kadmin-commands.in

```diff
@@ -415,6 +415,13 @@ command = {
 	max_args = "1"
 	help = "Modifies some attributes of the specified principal."
 }
+command = {
+	name = "prune"
+	argument = "principal prunekvno"
+	min_args = "2"
```

Comment:
Make the `prunkvno` argument optional, default it to zero.  See comments below.

Reply:
done (commit 161cb76)